src/main/java/frc/robot/Constants.java

```diff
@@ -18,9 +18,26 @@
  * wherever the constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final double defaultNull = Double.MIN_VALUE;
-    public static final double intakeSpeed = 0.5;
-    
+
+    public static final class ShooterConstants {
+        public static final int LEFT_FLYWHEEL_PORT = 16;
+        public static final int RIGHT_FLYWHEEL_PORT = 17;
+
+        public static final int KICKER_PORT = 26;
+        public static final int WHEELS_PORT = 27;
```

Comment:
Do we want to put ports in constants? I think just leaving them in hardwaremap would be fine, since it would make both files slightly shorter, and hardwaremap is essentially constants for hardware.

Reply:
I think they belong in Constants. Right now, the line is pretty clear that any constant numbers go in constants. If we move ports to hardwareMap, it'll blur that line and maybe cause confusion.

Reply:
I disagree, since I think just keeping ports out of constants wouldn't blur the line at all. However, I do see your point and it isn't that big a deal, so it's really not worth the extra effort to change everything.